pkg/chunk/chunk_store_test.go

```diff
@@ -14,22 +14,45 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/weaveworks/cortex/pkg/prom1/storage/local/chunk"
+	"github.com/weaveworks/cortex/pkg/util"
 	"github.com/weaveworks/cortex/pkg/util/extract"
 	"golang.org/x/net/context"
 
 	"github.com/weaveworks/common/test"
 	"github.com/weaveworks/common/user"
 )
 
+var schemas = []struct {
+	name              string
+	fn                func(cfg SchemaConfig) Schema
```

Comment:
I was expecting `storeFn` to come in this commit, even though they would all be the same at this point.